spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -311,7 +315,6 @@
           expect(job_analytics).to have_logged_event(
             'GetUspsProofingResultsJob: Exception raised',
             hash_including(
-              minutes_since_established: range_approximating(3.days.in_minutes, vary_right: 5),
               exception_message: error_message,
```

Comment:
```suggestion
              exception_message: error_message,
              exception_class: 'StandardError',
              reason: 'Request Exception',
```
could also test for exception class and reason here